src/Nexus.js

```diff
@@ -203,6 +203,14 @@ module.exports = function xGraph() {
 
 			}
 
+			function indirectEvalImp(entString) {
+				let imp = (1, eval)(entString);
+				console.dir(imp);
```

Comment:
Not sure we want be printing all of this out

Reply:
tru